plugins/rbac-backend/src/service/policy-builder.ts

```diff
@@ -65,11 +66,27 @@ export class PolicyBuilder {
       policy: await RBACPermissionPolicy.build(env.logger, env.config, enf),
     };
 
+    const pluginIds = env.config.getOptionalStringArray(
+      'permission.rbac.pluginsWithPermission',
+    );
+    if (pluginIds) {
+      // use plugin ids from application configuration.
+      env.pluginIdProvider = {
```

Comment:
Do you override the IDs returned by the  `pluginIdProvider` passed in argument, but the pluginIds provided in the config ?
Would it make sense to merge them, so that the config allows adding some, but not removing the ones passed in the code ?

Reply:
Ok. Done.